style.css

```diff
@@ -291,3 +413,268 @@ footer {
   justify-content: center;
   align-content: space-between;
 }
+
+@media all and (min-width: 992px) and (max-width: 1100px) {
+  body {
+    background: #191765;
```

Comment:
- [ ] You have done a good job, but there's still a problem with your design with width after 1100px. Kindly delete the `(max-width: 1100px)` and just leave the  `(min-width: 992px)`.

![image](https://user-images.githubusercontent.com/35535130/154743808-071f1c03-cabb-4c0d-b2da-1924b15e3db2.png)

![image](https://user-images.githubusercontent.com/35535130/154744145-2a0658af-4595-4358-8671-97aee52b0e14.png)